### ISSUE TITLE
Updating woltka in existing conda

### ISSUE THREAD

**Issue by adswafford** [qiyunzhu/woltka#27]
Just as a heads up, after activating my previous woltka conda environment, when I ran:

pip install git+https://github.com/qiyunzhu/woltka.git --upgrade

I got the error message:
Collecting git+https://github.com/qiyunzhu/woltka.git
  Cloning https://github.com/qiyunzhu/woltka.git to /tmp/pip-req-build-93q92ldy
  Running command git clone -q https://github.com/qiyunzhu/woltka.git /tmp/pip-req-build-93q92ldy
Collecting biom-format
  Downloading biom-format-2.1.8-1.tar.gz (11.7 MB)
     |████████████████████████████████| 11.7 MB 1.2 kB/s
    ERROR: Command errored out with exit status 1:
     command: /home/adswafford/miniconda3/envs/woltk/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-_go44khn/biom-format/setup.py'"'"'; __file__='"'"'/tmp/pip-install-_go44khn/biom-format/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-_go44khn/biom-format/pip-egg-info
         cwd: /tmp/pip-install-_go44khn/biom-format/
    Complete output (12 lines):
    Traceback (most recent call last):
      File "/tmp/pip-install-_go44khn/biom-format/setup.py", line 19, in <module>
        import numpy as np
    ModuleNotFoundError: No module named 'numpy'

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-_go44khn/biom-format/setup.py", line 21, in <module>
        raise ImportError("numpy must be installed prior to installing biom")
    ImportError: numpy must be installed prior to installing biom
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

I ran:
conda install numpy

and got:
Collecting git+https://github.com/qiyunzhu/woltka.git
  Cloning https://github.com/qiyunzhu/woltka.git to /tmp/pip-req-build-e35_0yaz
  Running command git clone -q https://github.com/qiyunzhu/woltka.git /tmp/pip-req-build-e35_0yaz
Collecting biom-format
  Using cached biom-format-2.1.8-1.tar.gz (11.7 MB)
    ERROR: Command errored out with exit status 1:
     command: /home/adswafford/miniconda3/envs/woltk/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-v1i2s44i/biom-format/setup.py'"'"'; __file__='"'"'/tmp/pip-install-v1i2s44i/biom-format/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-v1i2s44i/biom-format/pip-egg-info
         cwd: /tmp/pip-install-v1i2s44i/biom-format/
    Complete output (12 lines):
    Traceback (most recent call last):
      File "/tmp/pip-install-v1i2s44i/biom-format/setup.py", line 25, in <module>
        from Cython.Build import cythonize
    ModuleNotFoundError: No module named 'Cython'

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-v1i2s44i/biom-format/setup.py", line 27, in <module>
        raise ImportError("cython must be installed prior to installing biom")
    ImportError: cython must be installed prior to installing biom
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

So finally:
conda install cython

It isn't a huge issue, just some documentation notes for updating to the current version.

**Comment by qiyunzhu** [comment]
Hi @adswafford Thanks for testing the program and sharing your discovery! Previously I also realized that installation of BIOM is an issue in some computers. I found that running `conda install biom-format` before `pip install woltka` will solve the problem in some scenarios. Once Woltka becomes available in a conda channel, this issue should disappear. But for now, I think I will add an extra statement in the documentation to recommend `conda install biom-format`.
